db/db_compaction_test.cc

```diff
@@ -455,6 +455,61 @@ TEST_P(DBCompactionTestWithParam, CompactionDeletionTrigger) {
   }
 }
 #endif  // !defined(ROCKSDB_VALGRIND_RUN) || defined(ROCKSDB_FULL_VALGRIND_RUN)
+TEST_F(DBCompactionTest, UniversalBottomPriCompactionRepickNothing) {
+  const int kFileNumCompactionTrigger = 3;
+
+  Options options = CurrentOptions();
+  options.universal_pick_compaction_by_thread_pri = true;
```

Comment:
Can we add more tests with the new option? E.g. a full compaction is picked and executed when bottom threads are available to test the intent forwarding and compaction re-picking.

Reply:
To be fixed